### PR TITLE
feat: add metadata

### PR DESCRIPTION
**Pull request by yshrsmz**
part of #25 

I've added

- app_version
- os_version
- device_model

fields.

### CODE REVIEW

**Review by cre8ivejp — APPROVED** [review]
Great!
Thank you!

**Comment by yshrsmz** [comment]
@cre8ivejp 
Hi, I've updated androidTest test cases please take a look(just a minor changes)

[`e01abe3` (#35)](https://github.com/bucketeer-io/android-client-sdk/pull/35/commits/e01abe3b001130255d4efb3b09b1b8bef551fe42)

File: bucketeer/src/main/kotlin/io/bucketeer/sdk/android/internal/event/EventCreators.kt [hunk 1]
```diff
@@ -84,6 +98,7 @@ internal fun newGoalEvent(
       tag = featureTag,
       source_id = SourceID.ANDROID,
       sdk_version = BuildConfig.SDK_VERSION,
+      metadata = newMetadata(appVersion),
```
Comment (cre8ivejp) [inline bucketeer/src/main/kotlin/io/bucketeer/sdk/android/internal/event/EventCreators.kt:101]: I just noticed that I forgot to add the `metadata` also to MetricsEvent proto message. I will add it on Monday, but you can add it in this PR if you like and merge it later.

Comment (cre8ivejp) [reply bucketeer/src/main/kotlin/io/bucketeer/sdk/android/internal/event/EventCreators.kt:101]: @yshrsmz 
I have added it to this PR.
https://github.com/bucketeer-io/bucketeer/pull/132

Comment (yshrsmz) [reply bucketeer/src/main/kotlin/io/bucketeer/sdk/android/internal/event/EventCreators.kt:101]: I've added metadata to MetricsEvent in [`6aa09a9` (#35)](https://github.com/bucketeer-io/android-client-sdk/pull/35/commits/6aa09a94838705df6405f4de207925eff9b67472)

**Review by cre8ivejp — APPROVED** [review]
Nice!
Thank you!

**Comment by yshrsmz** [comment]
Thanks!

**Pull request merged**